app/models/miq_queue.rb

```diff
@@ -121,7 +121,7 @@ def self.put(options)
     options[:args] = [options[:args]] if options[:args] && !options[:args].kind_of?(Array)
 
     msg = MiqQueue.create!(options)
-    msg.warn_if_large_payload
+    msg.warn_if_large_payload unless Rails.env.production?
```

Comment:
Agreed, I don't understand why we need to pay the penalty of yaml dump and potentially logging in every  queue put. Even if we have the yaml string, let's figure out what this log message solves. Do we really care about the size or do we really care how long it takes to create or something else?